.github/workflows/test.yml

```diff
@@ -2,7 +2,28 @@
 
 name: CI
 
-on: [push, pull_request]
+on:
+  # Run on pushes to the default branch.
+  push:
+    branches:
+      - main
+
+  # Run on all PRs.
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+  # Support merge queues.
+  merge_group:
+
+  # Run on a schedule.
+  schedule:
+    - cron: "0 7 * * 1" # Every Monday at 9 in the morning CET
```

Comment:
I'm not sure running on a schedule is necessary. The tests are not generally flaky (except for the GitHub rate limits) so testing `main` + PRs should be good enough.

Reply:
> If there aren't flakes, I'm happy to get rid of it, but it might be useful as a reminder to update elm-syntax in tests.

Won't we get dependabot PRs for this anyway?

Reply:
Oh, yeah, I forgot I did that 🤦‍♂️